app/graphql/types/query_type.rb

```diff
@@ -20,6 +20,16 @@ def providers(filters: nil, order_by: nil)
       scope
     end
 
+    field :provider,
```

Comment:
I *think* this can just be:

```ruby
field :provider, null: false do
  argument :id, ID, required: true
end
def provider(id:)
  ::Provider.where(id: id.to_i)
end
```